src/window.c

```diff
@@ -25,9 +25,12 @@
  *   PLAYLIST
  */
 
+int current_index;
```

Comment:
I would rather create a get_index() function in vlcPlayer.c so that you do not have two parallel indices with same values working in two different files.

Reply:
thanks, I access vlc->media_index now directly.